Firestore/core/src/api/settings.h

```diff
@@ -97,7 +97,7 @@ class LocalCacheSettings {
   friend class Settings;
 
  public:
-  enum class Kind { kMemory, kPersistent };
+  enum class Kind { kMemory = 1, kPersistent };
```

Comment:
what is the motivation for adding enum values (`=1`) here and below? Is it needed? By default the compiler assigns values starting at 0.

Reply:
Without this, some equality test on settings fail. I did this PR long time ago, so cannot recall the details.